infinispan-remote/infinispan-secured-keycloak/docker-compose.yaml

```diff
@@ -13,7 +13,7 @@ services:
     network_mode: host
 
   infinispan-server:
-    image: quay.io/infinispan-test/server:14.0.x
+    image: quay.io/infinispan/server:15.0
```

Comment:
Shouldn't this be 14.0? The rest of the `main` branch still seems to be based upon 14.x

Reply:
both work, but the console is updated in 15 with access management. I rather leave 15.0 here even if it's still dev for a couple of weeks

Reply:
What happens when we create the 14.0.x branch from `main` though, it will be inconsistent if this is on 15.0.

I would suggest we merge as `14.0` into `main` and then we create the `14.0.x` branch and update `main` to use 15.0.x throughout as this is required sooner rather than later anyway.

Reply:
in any case, merge this one, and i'll to that just after plz

Reply:
ah ok I understood. nvm